.github/workflows/pr-description-update.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           delimiter="Changelog picked up from commits here:"
           cleaned_body="${pr_body%${delimiter}*}"
           cleaned_body_without_trailing_space="$(python .github/remove_trailing_space.py "${cleaned_body}")"
-          conventional_commits="$(git log --pretty=format:"%B"  origin/main..HEAD | grep -Pio '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test|¯\\_\(ツ\)_\/¯)(\(\w+\)?((?=:\s)|(?=!:\s)))?(!)?(:\s.*)?(\n\n.*)?$' | grep ':' || true)"
+          conventional_commits="$(git log --pretty=format:"%B"  origin/${{github.base_ref}}..HEAD | grep -Pio '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test|¯\\_\(ツ\)_\/¯)(\(\w+\)?((?=:\s)|(?=!:\s)))?(!)?(:\s.*)?(\n\n.*)?$' | grep ':' || true)"
```

Comment:
I'm adding this because otherwise was picking all the conventional commits from the main branch and in this case we have the `breaking-release` as a base branch

Reply:
Nice!